frontend/app/templates/components/footer-main.hbs

```diff
@@ -1,4 +1,4 @@
-{{!-- <footer>
```

Comment:
Why was the specification removed?

Reply:
@abishekvashok  This specification comments the code. I have just reverted the commented code with adding other features and scroll able footer.